src/transformers/configuration_pegasus.py

```diff
@@ -0,0 +1,62 @@
+""" PEGASUS model configuration """
+
+import logging
+
+from .configuration_bart import BartConfig
+
+
+logger = logging.getLogger(__name__)
+
+
+class PegasusConfig(BartConfig):
```

Comment:
Don't think we should abstract from `BartConfig` just to avoid the `self.....` statements. It's great to have the config as a stand-alone file to directly see all config logic. Abstraction does not add much functionality here IMO.

Reply:
I tried to strike a balance of having the defaults be obvious and readable without allowing the config to ever be different from bart's.  There is no config logic to see really, just values.

Reply:
The defaults are also shown nicely on the `pegasus.rst` page.